tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -2173,11 +2246,19 @@ class MklQuantizedConvOp
     }
     if (!is_bias_const_ || IsBiasCacheEmpty(context) || !scales_are_valid) {
       dnnl::primitive_attr bias_attr;
+#ifndef ENABLE_ONEDNN_V3
       if (depth == 1) {
         bias_attr.set_output_scales(0, scales_);
       } else {
         bias_attr.set_output_scales(1, scales_);
       }
+#else
+      // TODO(intel-tf): Enable this for int8 when using oneDNN v3.x
+      assert(depth == 1);
```

Comment:
Same here. Could you please replace `assert` with `DCHECK` and add a TODO to replace `DCHECK` with returning an error status?

Reply:
Done.